curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f93b67169a9c703264458a.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 65f93b67169a9c703264458a
+title: Step 30
+challengeType: 20
+dashedName: step-30
+---
+
+# --description--
+
+Since the method should returned the string to instantiate the object for `R2Vector` as well as `R3Vector` when inherited, you cannot build the string specifying the class name.
```

Comment:
```suggestion
Since the method should return the string to instantiate the object for `R2Vector` as well as `R3Vector` when inherited, you cannot build the string specifying the class name.
```